meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -328,6 +328,9 @@ PREFERRED_PROVIDER_virtual/dtb_imx8mmevk ?= "lmp-device-tree"
 SOTA_CLIENT_FEATURES_append_imx8mmevk = " ubootenv"
 MACHINE_FEATURES_remove_imx8mmevk = "qca9377 qca6174"
 UBOOT_MACHINE_imx8mmevk = "imx8mm_evk_config"
+# Secondary Boot, SIT offset is 0x1000 (4096 in decimal)
```

Comment:
I'm going to start with: naming is hard

But this comment is mildly confusing.  SIT (used as an abbreviation) almost always references the table flashed to sector 65 (0x8200 byte offset).  And SIT offset is confusing in that context.

Then the naming of the OE vars below is just generic enough to also be confusing.

Maybe we can standardize to the same naming as the u-boot `CONFIG_SECONDARY_BOOT_SECTOR_OFFSET` (dropping the ENABLE var):
```
# Enable secondary boot sector offset
IMXBOOT_SECONDARY_BOOT_SECTOR_OFFSET_imx8mmevk = "4096"
```

Reply:
If we can automatically find out if secondary is enabled on u-boot, and find the offset as well, then this would be preferred. My only concern is the extra work required to find that out, but I think it is still better than maintaining changes in lmp-machine-custom as that costs later when pushing to factories.